lib/vmpooler/api/v1.rb

```diff
@@ -162,6 +162,41 @@ def need_token!
       JSON.pretty_generate(result)
     end
 
+    get "#{api_prefix}/token/?" do
+      content_type :json
+
+      status 404
+      result = { 'ok' => false }
+
+      if Vmpooler::API.settings.config[:auth]
+        status 401
+
+        need_auth!
+
+        backend.keys('vmpooler__token__*').each do |key|
```

Comment:
Dev facing docs on the data model in Redis, or an abstraction over it, would be awesome for coming up to speed on pooler development.

IOW, from elsewhere in this file this _looks_ right. But I don't know what I don't know, ya know?